rero_ils/dojson/cli.py

```diff
@@ -17,6 +17,7 @@
 
 """Click command-line utilities."""
 
+import json
```

Comment:
why?

Reply:
This is to correct a bug when generating the json file.